mesa/visualization/__init__.py

```diff
@@ -6,3 +6,7 @@
 TextServer: Class which takes a TextVisualization child class as an input, and
 renders it in-browser, along with an interface.
 '''
+
+from . import CanvasServer
```

Comment:
These are import to initialize the module to trigger tests, but also is what is causing the flake8 error.